var/spack/repos/builtin/packages/hipsolver/package.py

```diff
@@ -145,6 +146,7 @@ def cmake_args(self):
         ]
 
         args.append(self.define_from_variant("USE_CUDA", "cuda"))
+        args.append(self.define_from_variant("BUILD_FORTRAN_BINDINGS", "build_fortran_bindings"))
```

Comment:
I wouldn't bother with the variant. `BUILD_FORTRAN_BINDINGS` only affects the tests. It ultimately just controls whether the hipsolver tests will check the Fortran bindings. I think you can just set it to `OFF` for the Spack package.